static/app/utils/profiling/renderers/spansRenderer.tsx

```diff
@@ -0,0 +1,55 @@
+import {mat3} from 'gl-matrix';
+
+import {Rect, resizeCanvasToDisplaySize} from 'sentry/utils/profiling/gl/utils';
+import {SpanChart, SpanChartNode} from 'sentry/utils/profiling/spanChart';
+
+// Convert color component from 0-1 to 0-255 range
+function colorComponentsToRgba(color: number[]): string {
+  return `rgba(${Math.floor(color[0] * 255)}, ${Math.floor(color[1] * 255)}, ${Math.floor(
+    color[2] * 255
+  )}, ${color[3] ?? 1})`;
+}
+
+export class SpanChartRenderer2D {
```

Comment:
this is a very simple an inefficient renderer, the point is to get the spans drawing on the screen first, optimize this later (if we have to)